### PR TITLE
chore: add arch diagrams - 3.1

### PR DESCRIPTION
**Pull request by askmeegs**

### CODE REVIEW

**Comment by askmeegs** [comment]
Closing this until we know the slug of the jump start solution in the Cloud Console. that will determine the file names.

**Review by NimJay — APPROVED** [review]
Not that my approval is necessary here, but as the [JSS diagram lead](http://go/jss-diagrams), I approve. :)

File: images/architecture.json
Comment (NimJay) [inline images/architecture.json:1]: **Nitpick:** Looks like the linter is complaining about the JSON file: `Error: No newline at end of file ./images/architecture.json`.

Comment (askmeegs) [reply images/architecture.json:1]: I ran a `format` on the JSON file which added a newline, hope that fixed it.

**Review by balajismaniam — APPROVED** [review]
LGTM and merging as the lint failures were already existing (should be handled separately).

**Pull request merged**